### ISSUE TITLE
After global approximately step=7000 the loss is enormous for specific steps 

### ISSUE THREAD

**Issue by Evaggelou** [tensorflow/models#4847]
System information
What is the top-level directory of the model you are using: models/research
Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No, only slightly modified (config, some syntax conversion from python2 to python3.....)
OS Platform and Distribution (e.g., Linux Ubuntu 16.04):ubuntu 16.04
TensorFlow installed from (source or binary): tensorflow ubuntu (venv, CPU support only)
TensorFlow version (use command below): v1.9.0-0-g25c197e023 1.9.0
Bazel version (if compiling from source): N/A
CUDA/cuDNN version: N/A
CPU/GPU model and memory: GeForce GTX 1080 Graphics Card with 10gb
Exact command to reproduce: python3 train.py --logtostderr --train_dir=/home/spacy03/evangelou_test/models-master/research/object_detection/training/faster_rcnn_inception_v2_coco_2018_01_28 --pipeline_config_path=/home/spacy03/evangelou_test/models-master/research/object_detection/faster_rcnn_inception_v2_coco_2018_01_28/pipeline.config

Describe the problem
I cannot find any similar issues from anywhere, after 7000 step the loss for some step become very big.  It becomes bigger and bigger. I cannot find a pattern in steps and it doesn;t make sense the data are splited balanced

10 classes
train data:10 images for every class(i try a specific experiment that's why)
eval data:60 images for every class
Also i tried to put more data to train and i still have the same problem.

![screenshot from 2018-07-20 11-47-04](https://user-images.githubusercontent.com/18635146/42993182-6458213e-8c13-11e8-8b8e-98bfe65f36e9.png)

The configuration file:
model {
  faster_rcnn {
    num_classes: 10
    image_resizer {
      keep_aspect_ratio_resizer {
        min_dimension: 600
        max_dimension: 1024
      }
    }
    feature_extractor {
      type: "faster_rcnn_inception_v2"
      first_stage_features_stride: 16
    }
    first_stage_anchor_generator {
      grid_anchor_generator {
        height_stride: 16
        width_stride: 16
        scales: 0.25
        scales: 0.5
        scales: 1.0
        scales: 2.0
        aspect_ratios: 0.5
        aspect_ratios: 1.0
        aspect_ratios: 2.0
      }
    }
    first_stage_box_predictor_conv_hyperparams {
      op: CONV
      regularizer {
        l2_regularizer {
          weight: 0.0
        }
      }
      initializer {
        truncated_normal_initializer {
          stddev: 0.00999999977648
        }
      }
    }
    first_stage_nms_score_threshold: 0.0
    first_stage_nms_iou_threshold: 0.699999988079
    first_stage_max_proposals: 100
    first_stage_localization_loss_weight: 2.0
    first_stage_objectness_loss_weight: 1.0
    initial_crop_size: 14
    maxpool_kernel_size: 2
    maxpool_stride: 2
    second_stage_box_predictor {
      mask_rcnn_box_predictor {
        fc_hyperparams {
          op: FC
          regularizer {
            l2_regularizer {
              weight: 0.0
            }
          }
          initializer {
            variance_scaling_initializer {
              factor: 1.0
              uniform: true
              mode: FAN_AVG
            }
          }
        }
        use_dropout: false
        dropout_keep_probability: 1.0
      }
    }
    second_stage_post_processing {
      batch_non_max_suppression {
        score_threshold: 0.300000011921
        iou_threshold: 0.600000023842
        max_detections_per_class: 100
        max_total_detections: 100
      }
      score_converter: SOFTMAX
    }
    second_stage_localization_loss_weight: 2.0
    second_stage_classification_loss_weight: 1.0
  }
}
train_config {
  batch_size: 1
  data_augmentation_options {
    random_horizontal_flip {
    }
  }
  optimizer {
    momentum_optimizer: {
      learning_rate: {
        manual_step_learning_rate {
          initial_learning_rate: 0.0003
          schedule {
            step: 900000
            learning_rate: .0003
          }
          schedule {
            step: 1200000
            learning_rate: .000003
          }
        }
      }
      momentum_optimizer_value: 0.9
    }
    use_moving_average: false
  }
  gradient_clipping_by_norm: 10.0
  fine_tune_checkpoint: "/home/spacy03/evangelou_test/models-master/research/object_detection/faster_rcnn_inception_v2_coco_2018_01_28/model.ckpt"
  from_detection_checkpoint: true
  num_steps: 200000
}
train_input_reader {
  label_map_path: "/home/spacy03/evangelou_test/models-master/research/object_detection/training/label_map.pbtxt"
  tf_record_input_reader {
    input_path: "/home/spacy03/evangelou_test/models-master/research/object_detection/train.record"
  }
}
eval_config {
  num_examples: 600
  use_moving_averages: false
}
eval_input_reader {
  label_map_path: "/home/spacy03/evangelou_test/models-master/research/object_detection/training/label_map.pbtxt"
  shuffle: true
  num_readers: 1
  tf_record_input_reader {
    input_path: "/home/spacy03/evangelou_test/models-master/research/object_detection/test.record"
  }
}

**Comment by kulsemig** [comment]
Hi, @Evaggelou. Have you solved your problem? How?
